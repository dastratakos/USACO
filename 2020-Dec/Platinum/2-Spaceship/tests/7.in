60 5 60
111011011100011110010111110101001001111011101110001011010001
111010011110111101001111011000111111111110111011000010100101
110110000001110111101100001001111111101100010001000100100101
110110001011111101111111011001011111010110111110111111000101
111010000100011110101111111011101111110111101001110011011111
100011111111111101100111011011011110010010011111011100111111
111011000011010011100111110100111010011011111101000011111110
111011000110101011111111001110010111101011011110101100110111
110000111111101101010011110101001110111111011110111100111100
001111110110000011011011100101101111110101011111011111111101
111001111110100010111010100111101101110111011100111111110110
111000011110111110101111101101111111001110001111101110110100
100100111111111101001110011010111110001111001111110100000101
111111111001111100101100101001111111110111101000110101101011
100101011011110000010111111011111101101000010000001001100011
111111100000111110111010111111101010111111100011011011100111
001110010110111011100001111101101101111010000101011010111101
110111101111111101111100110100111011111011111111000110111100
011010100110111111100101111101001001011111110111011011110011
111111110111110111011111111110111011000100000111111111101001
100111111111001111011111111011111111100001111110111110101101
111111111111111001101000111111100111000110000011001111010111
100100001101011110110011101110111110111111110111001010111111
111011111111100100101001111001001111101011010011011111100110
001101101101001111110110001111001011010100111111111111100110
000111110010110111000001101101101101111010101000011011101010
001100101111111111011011101000111010001011011111111000110110
000001110111111110111110100100110111100111110101111110110111
001110110010101100001101001111101111101111011111111101010111
111110101111000111111111111011101101111111111101110100101111
110111110100111100011010011010101110111111111111101111101011
000110111010111010011001011111011101000011110111111111101110
001000001001011101010011101101001111111110001101111111111001
110101011111110101111001000111111111111111011111111011111111
111111111111101111001111010110001101011100101010001101111011
110100111110100111100110111100111011111110111011101101101111
100110110010100110110001011110111111111110110111011111101110
111111011111101110101011001111110111111011111111000001010111
111111111001110111101101001101011011001110111011101111111000
111100110010111101110111100110001001000110110101111110111101
011101011111110001101001110101111011011100010111010010001101
011011110110011011111110110111001101100010111011101101111111
011110011111010010101011110101101110010000011001111000101011
011000011000000001111110101111111011010111111000001011011101
110010111101011101101111010101111110111110011011101110011111
111101101110111111110011110011111011111111110111101111001101
110111111101110101001111011001011101000101110111111110111010
000110111101111001111110011001111010001010110110011101110010
111110011111010011101110101110011111011111101010010010101111
010001011101101011101010111101101111111011101111100101101111
110111111111111110111011111010011101111110101110001101001100
011111011111111100011110100100101111011110111011101101101101
001110010100111111111011100100011000111000111110111111110110
000100111111100110100011100111100111101101101000010111011011
110000000111100111111111011111111000011111010101101011111011
100110111110111111101100000111111110101110110011000101001001
111000110100110100101010010000011011111010111110110101001110
001011111110001111101101111101111101011011101101100111110101
011110111110001111111011001110110111101110010111111110111101
011001101110111111101101011011110001010100010110110101010101
3 5 5 1
3 5 3 54
3 5 4 54
3 5 5 2
3 5 5 40
3 5 2 55
3 5 5 8
3 5 1 50
3 5 5 31
3 5 1 3
3 5 4 46
3 5 3 18
3 5 1 55
3 5 1 39
3 5 2 18
3 5 2 39
3 5 2 14
3 5 5 33
3 5 4 1
3 5 2 40
3 5 3 34
3 5 4 45
3 5 4 8
3 5 1 50
3 5 1 49
3 5 5 33
3 5 1 13
3 5 4 51
3 5 4 56
3 5 4 46
3 5 4 27
3 5 4 23
3 5 2 49
3 5 3 30
3 5 1 15
3 5 1 59
3 5 2 5
3 5 4 44
3 5 4 48
3 5 1 10
3 5 5 35
3 5 4 40
3 5 5 59
3 5 5 12
3 5 1 15
3 5 1 26
3 5 1 19
3 5 2 25
3 5 3 16
3 5 2 12
3 5 4 22
3 5 2 14
3 5 5 44
3 5 3 14
3 5 5 51
3 5 1 13
3 5 5 45
3 5 4 44
3 5 5 34
3 5 3 5
