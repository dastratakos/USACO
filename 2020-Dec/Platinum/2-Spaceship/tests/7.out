903762175
620466655
742247772
475706978
98326678
764555675
228287236
326479687
252867412
247200100
297651745
898381695
480984894
496975149
702263844
614947858
888139714
515449503
533453995
645476093
298210024
476837183
990748610
326479687
894031168
515449503
659439388
293335974
139383383
297651745
114643765
891375100
59637399
802722814
74977144
347862832
329694326
757977281
505063814
361441433
970615227
932849015
997989486
800444947
74977144
750924256
499156110
569477184
566367503
523266283
178135231
888139714
795616520
335344115
899440583
659439388
425429656
757977281
132113687
172893900
